package.sh

```diff
@@ -17,16 +17,28 @@ echo "version=$version"
 
 win_export_path="project/export/windows"
 win_zip_filename="$win_export_path/turbofat-win-v$version.zip"
+win_bat_filename="$win_export_path/turbofat-win-troubleshoot-v$version.bat"
+template_filename="bin/troubleshoot_bat/turbofat-troubleshoot.bat.template"
```

Comment:
Should this be called something like `bat_template_filename`? We have export templates, and this could be ambiguous.

Reply:
Changed to `win_bat_template`.